app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -71,7 +71,7 @@ def update
 
     def create
       if create_admin_set
-        redirect_to hyrax.edit_admin_admin_set_path(@admin_set), notice: I18n.t('new_admin_set', scope: 'hyrax.admin.admin_sets.form.permission_update_notices', name: @admin_set.title.first)
+        redirect_to hyrax.edit_admin_admin_set_path(admin_set_id), notice: I18n.t('new_admin_set', scope: 'hyrax.admin.admin_sets.form.permission_update_notices', name: @admin_set.title.first)
```

Comment:
This works for both `AdminSet` (active fedora) and `Hyrax::AdministrativeSet` (valkyrie).  The method gets the id and forces it to a `String`.  The redirect fails when id is a `Valkyrie::ID`